Makefile

```diff
@@ -50,9 +50,9 @@ $(OUT)/cpu_instrs.h: tests/cpu_instrs.gb tests/rom2h.c
 	$(Q)$(CC) -o $(OUT)/rom2h tests/rom2h.c
 	@$(OUT)/rom2h
 
-$(OUT)/bench: $(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h
+$(OUT)/bench: gbit/lib/tester.c gbit/lib/ref_cpu.c gbit/lib/inputstate.c gbit/lib/disassembler.c
```

Comment:
You have to ensure the availability of `gbit`. That is, you might execute `git submodule update --init`.